app/controllers/admin/tags_controller.rb

```diff
@@ -2,6 +2,12 @@ module Admin
   class TagsController < Admin::ApplicationController
     layout "admin"
 
+    ALLOWED_PARAMS = %i[
```

Comment:
Not directly related to this bugfix, but since I was here already I moved the allowed params into a constant.